cpp_design_bookmarks.md

```diff
@@ -466,7 +466,7 @@ C++ Exception Specifications
 -
 
 The Herb Sutter's article ["A Pragmatic Look at Exception Specifications"](http://www.gotw.ca/publications/mill22.htm) ([[pl@es]](https://github.com/kuzminrobin/code_review_notes/blob/master/article_list.md)) makes think that the exception specifications  
-* instead of providing the _compile-time_ nothrow-guarantee (`throw()`) or throws-only-these-ones-gaurantee (`throw(A, B)`)  
+* instead of providing the _compile-time_ nothrow-guarantee (`throw()`) or throws-only-these-ones-guarantee (`throw(A, B)`)  
 * provide the _run-time_ checks/asserts (which are _pessimization_, i.e. they lower down the performance).  
 
 Another confirmation is in [[eses]](https://github.com/kuzminrobin/code_review_notes/edit/master/article_list.md).  
```

Comment:
@jwilk, thank you very much for the contribution! This is the first-most contribution to my files. I'm still to study [how contributing works](https://github.com/kuzminrobin/code_review_notes/community) on GitHub (I'm a newbie). Feel free to let me know if I'm doing anything wrong.
I agree with your changes and I'm about to merge them. If I'm expected to do anything different (e.g. just to approve the changes and give you the opportunity to merge or rebase+merge those) then again feel free to let me know.
Thank you, your participation inspires...